plugins/backstage-plugin-env0/src/components/env0-variables-input/env0-variable-field.tsx

```diff
@@ -58,12 +58,28 @@ export const doesVariableValueMatchRegex = (variable: Variable) => {
   return regex.test(variable.value || '');
 };
 
+const getVariableHelperText = (
+  isRegexWrong: boolean,
+  isRequiredValueMissing: undefined | boolean,
+  variable: VariableFields,
+) => {
+  if (isRegexWrong) {
+    return `Value does not match regex: ${variable.regex}`;
+  }
+  if (isRequiredValueMissing) {
+    return 'Value is required';
+  }
+  return '';
+};
+
```

Comment:
this is similar to the code in `use-variables-validation`?
export or use same logic, no?

Reply:
texts are a bit different since this is what we show under the variable and the validation errors are shown at the top of the form....